### PR TITLE
Log errors in work peer client

### PR DESCRIPTION
**Pull request by cryptocode**
Also fix log output where port was reported as an error code

### CODE REVIEW

File: rai/node/node.cpp [hunk 1]
```diff
@@ -2176,27 +2176,27 @@ class distributed_work : public std::enable_shared_from_this<distributed_work>
...
-											BOOST_LOG (this_l->node->log) << boost::str (boost::format ("Unable to read from work_peer %1% %2%") % connection->address % connection->port);
+											BOOST_LOG (this_l->node->log) << boost::str (boost::format ("Unable to read from work_peer %1% %2%: %3%") % connection->address % connection->port % ec);
 											this_l->failure (connection->address);
 										}
 									});
 								}
 								else
 								{
-									BOOST_LOG (this_l->node->log) << boost::str (boost::format ("Unable to write to work_peer %1% %2%") % connection->address % connection->port);
+									BOOST_LOG (this_l->node->log) << boost::str (boost::format ("Unable to write to work_peer %1% %2%: %3%") % connection->address % connection->port % ec);
```
Comment (clemahieu) [inline rai/node/node.cpp]: Do we want to see the code or ec.message()?

Comment (cryptocode) [reply rai/node/node.cpp]: Good question. Not sure if a message is always present, so maybe both? Error message with code in parenthesis sound good?

Comment (clemahieu) [reply rai/node/node.cpp]: Oh yea, that's a good idea.

Comment (cryptocode) [reply rai/node/node.cpp]: Updated

**Pull request merged**